config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users do
+    resources :users, only: [:index, :show]
+    resources :posts, only: [:index, :show]
+  end
```

Comment:
- You did well implementing the optional change requested by the reviewer, but as me mentioned, the route should be:
```
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
end 

```